lib/components/form/call-taker/date-time-options.js

```diff
@@ -190,40 +190,36 @@ export default class DateTimeOptions extends Component {
             <option key={o.value} {...o} />
           ))}
         </select>
-        <span style={{display: 'inline-flex'}}>
-          <OverlayTrigger
-            overlay={<Tooltip id='time-tooltip'>{cleanTime}</Tooltip>}
-            placement='bottom'
-            trigger={['focus', 'hover']}
-          >
-            <input
-              className='datetime-slim'
-              onChange={this.handleTimeChange}
-              onFocus={this.handleTimeFocus}
-              onKeyDown={onKeyDown}
-              style={{
-                fontSize: 'inherit',
-                lineHeight: '.8em',
-                marginLeft: '3px',
-                padding: '0px',
-                width: '50px'
-              }}
-              value={timeInput || dateTime.format('H:mm')}
-            />
-          </OverlayTrigger>
-        </span>
+        <OverlayTrigger
+          overlay={<Tooltip id='time-tooltip'>{cleanTime}</Tooltip>}
+          placement='bottom'
+          trigger={['focus', 'hover']}
+        >
+          <input
+            className='datetime-slim'
+            onChange={this.handleTimeChange}
+            onFocus={this.handleTimeFocus}
+            onKeyDown={onKeyDown}
+            style={{
+              fontSize: 'inherit',
+              lineHeight: '.8em',
+              marginLeft: '3px',
+              padding: '0px',
+              width: '50px'
+            }}
+            value={timeInput || dateTime.format('H:mm')}
+          />
+        </OverlayTrigger>
         <input
           className='datetime-slim'
           onChange={this.handleDateChange}
           onKeyDown={onKeyDown}
           style={{
             border: 'none',
```

Comment:
With the latest round of visual changes, could the date selector get a border too? (if so, make this control 2px wider for Firefox, and the `Plan` button needs a new height property)
![image](https://user-images.githubusercontent.com/56846598/128192371-edb3fc50-1173-4566-9760-c01396a84390.png)